df.map.xml

```diff
@@ -313,7 +320,7 @@
             </static-array>
         </static-array>
 
-        <static-array name='unk13' count='16'>
+        <static-array name='lightning' count='16'>
```

Comment:
is `lightning` correct? i can't find this in the bay12 headers. i think this might be intended to be `lighting`

Reply:
yes, you're right. this is a transcription error